presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueMetastoreModule.java

```diff
@@ -57,4 +66,16 @@ protected void setup(Binder binder)
                     .as(generator -> generator.generatedNameOf(GlueHiveMetastore.class));
         }
     }
+
+    @Provides
+    @Singleton
+    @ForGlueHiveMetastore
+    public Executor createCachingHiveMetastoreExecutor(HiveCatalogName catalogName, GlueHiveMetastoreConfig hiveConfig)
```

Comment:
This names seems wrong, since it's not for caching. Could just be `createExecutor` since the scope is for the Glue metastore module.

Reply:
Sure thing.

Reply:
This still needs updating

Reply:
Sorry for missing that one.